Dockerfile

```diff
@@ -1,7 +1,7 @@
-ARG PHP_VERSION=8.3
+ARG PHP_VERSION=8.4
 ARG ALPINE_VERSION=3.18
 
-FROM alpine:3.18.4 as sphinx-lint
+FROM alpine:3.18.4 AS sphinx-lint
```

Comment:
can go as separated PR and get auto-reviewe+merged

Reply:
#8379